src/game/TimeController.ts

```diff
@@ -43,22 +50,45 @@ export class TimeController {
   }
 
   get allowedMove() {
-    return this.realTime < this.stopAllowedMoveTime;
+    return this.cycleTime < this.stopAllowedMoveTime;
   }
 
   get safePeriod() {
-    return this.realTime > this.stopAllowedMoveTime && this.realTime < this.stopSafePeriod;
+    return this.cycleTime > this.stopAllowedMoveTime && this.cycleTime < this.stopSafePeriod;
   }
 
   get timeout() {
-    return this.realTime > this.stopSafePeriod && this.realTime < this.stopTimeout;
+    return this.cycleTime > this.stopSafePeriod && this.cycleTime < this.stopTimeout;
   }
 
   get timeOver() {
-    return this.realTime > this.stopTimeout;
+    return this.cycleTime > this.stopTimeout;
+  }
+
+  get endTime() {
```

Comment:
❗️
Это предикат, название должно быть соответствующим
```suggestion
  get timeEnded() { // или как-то так
```